satellite/gc/gc_test.go

```diff
@@ -109,6 +112,32 @@ func TestGarbageCollection(t *testing.T) {
 	})
 }
 
+func TestService_NewPieceTracker_and_Send(t *testing.T) {
+	config := gc.Config{
+		Interval:          time.Minute,
+		Active:            true,
```

Comment:
should update to Enabled

Reply:
Done by the merge of the last update of the base branch.